src/graphic/style_manager.cc

```diff
@@ -118,6 +118,21 @@ RGBAColor read_rgba_color(const LuaTable& table) {
 	return RGBAColor(rgbacolor[0], rgbacolor[1], rgbacolor[2], rgbacolor[3]);
 }
 
+// Read optional bool value from LuaTable
+bool read_bool(const LuaTable& table, const std::string& key) {
```

Comment:
These functions should take the default value as a third argument for readability

Reply:
I started to change them, but now I'm not sure it's an improvement. They're just shorthand convenience functions, so adding another option that will always have the same (quite obvious) value is contrary to the original purpose. These are only used by the next 2 functions, they're not even prototyped in the header.

Reply:
… unless you mean adding them to `lua_table.h`.